21.1 mz dashboards eud (#135)
DevExpress/dotnet-eud: dashboard-for-web/articles/web-dashboard-designer-mode/data-shaping/filtering.md

# Filtering

> Web Dashboard allows you to filter data in the [dashboard items](../dashboard-item-settings.md) or apply filters to a specific measure. [Dimensions](../bind-dashboard-items-to-data/bind-dashboard-items-to-data-in-the-web-dashboard.md) and [hidden dimensions](../bind-dashboard-items-to-data/hidden-data-items.md) are used to build filter criteria.

Reviewer: ```suggestion
Web Dashboard allows you to filter data in the [dashboard items](../dashboard-item-settings.md) or apply filters to a specific measure. You can use [dimensions](../bind-dashboard-items-to-data/bind-dashboard-items-to-data-in-the-web-dashboard.md) and [hidden dimensions](../bind-dashboard-items-to-data/hidden-data-items.md) to build filter criteria.
```

![web-filter-visible-data-filter-grids](../../../images/web-filter-visible-data-filter-grids.png)

> ## OLAP Filtering Specifics

Reviewer: ```suggestion
## OLAP Filter Specifics
```
